README.rst

```diff
@@ -1,50 +1,114 @@
-
-Project Az is built on Python (2.7, 3.4 and 3.5).
+A great cloud needs great tools; we're excited to introduce *Azure CLI 2.0 - Preview*, our next generation multi-platform command line experience for Azure.
 
 Installation
-============
+===============
+
+For OSX and Linux, you can install with the following curl script.
+
+.. code-block:: console
 
-For installation steps for common platforms, please take a look at our `preview installation guide <http://github.com/Azure/azure-cli/blob/master/doc/preview_install_guide.md>`__.
+   $ curl -L https://aka.ms/ProjectAzInstall | sudo bash
 
-Docker Setup (optional)
------------------------
-We have automated Docker images of the latest code in the master branch.
+We also maintain a docker image preconfigured with the Azure CLI.
 
-If you have not previously done so, `configure your Docker client engine <https://docs.docker.com/engine/installation/>`__.
+.. code-block:: console
 
-Then:
- + Run :code:`docker run -it azuresdk/azure-cli-python`
+   $ docker run -it azuresdk/azure-cli-python
 
-Alternatively:
- + Run :code:`alias az='docker run --rm -i -t -v ~/.azure:/root/.azure azuresdk/azure-cli-python az'`
- + Then run :code:`az` as normal. e.g. :code:`az account login`
+For installation steps for common platforms, as well as dependency troubleshooting, please take a look at our `installation guide <http://github.com/Azure/azure-cli/blob/master/doc/preview_install_guide.md>`__.
 
 Usage
 =====
-    
 .. code-block:: console
 
     $ az [ group ] [ subgroup ] [ command ] {parameters}
 
-For sample scripts and commands, please visit the `Demo Scripts <https://github.com/Azure/azure-cli/blob/master/doc/preview_demo_scripts.md>`__ page.  
 
-Download Source Code
-====================
+Getting Started
+=====================
+
+After installation, use the ``az configure`` command to help setup your environment and get you logged in.
+
+.. code-block:: console
+
+   $ az configure
+
+For usage and help content, pass in the ``-h`` parameter, for example:
+
+.. code-block:: console
+
+   $ az storage -h
+   $ az vm create -h
+
+Highlights
+===========
+
+Here are a few features and concepts that can help you get the most out of the Azure CLI 2.0 Preview
+
+Tab Completion
+^^^^^^^^^^^^^^
 
-To get the source code of the SDK via **git** type
+We support tab-completion for groups, commands, and some parameters
 
 .. code-block:: console
-    
-    $ git clone https://github.com/Azure/azure-cli.git
 
+   # looking up resource group and name
+   $ az vm show -g [tab][tab]
+   AccountingGroup   RGOne  WebPropertiesRG
+   $ az vm show -g WebPropertiesRG -n [tab][tab]
+   StoreVM  Bizlogic
+   $ az vm show -g WebPropertiesRG -n Bizlogic
 
+Querying
+^^^^^^^^
+
+You can use the ``--query`` parameter and the JMESPath query syntax to customize your output.
+
+.. code-block:: console
+
+   $ az vm list --query '[].{name:name,os:storageProfile.osDisk.osType}'
```

Comment:
All the examples use table format but table format isn't mentioned.

Consider adding `-otable` to the commands so it's obvious that you're using table format.
Or, have a section describing the table formats and say 'the example below use table format'.

Thoughts?